es/es-repository-hdfs/build.gradle

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+description = 'The HDFS repository plugin adds support for Hadoop Distributed File-System (HDFS) repositories.'
+def classname = 'org.elasticsearch.repositories.hdfs.HdfsPlugin'
+def plugin_repositoryhdfs_version = '1.0.0'
+
+configurations {
+    extraLibs.transitive = false
+}
+
+dependencies {
+    compile project(':es:es-core')
+    compile "org.apache.hadoop:hadoop-client:${versions.hadoop2}"
```

Comment:
we are now requiring all the hadoop deps? we should avoid that and as all are defined under the extraLib configuration it is not needed afail